athlinks-schema.ttl

```diff
@@ -174,4 +193,107 @@ athlinks:chipTime
 athlinks:agePercentage
   a owl:DataTypeProperty ;
   rdfs:domain athlinks:IntervalResult;
-  rdfs:range schema:Decimal .
+  rdfs:range xsd:decimal .
+
+athlinks:isHidden
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
+
+athlinks:isVisible
+  a owl:DataTypeProperty ;
+  rdfs:domain athlinks:AthleticRace;
+  rdfs:range xsd:boolean .
```

Comment:
These two names come up all over the place.  Unless we are willing to interpret the data, that is, we would have to reverse the value, and what if it is null?  What is the problem with just taking it as is?  CS_IsVisible comes up in athlinks race table, while is_hidden comes up in CTLive reg_option, for example.  It is not clear what the implication would be for software that depends on this value.  I think we should leave it.